src/controllers/userController.js

```diff
@@ -0,0 +1,76 @@
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  res.send(userService.getAllUsers());
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+
+  const user = userService.getUserById(id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const update = (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+  const user = userService.updateUser({ id, name });
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
```

Comment:
You are checking the type of 'name' after trying to update the user. This check should be done before attempting to update the user to avoid unnecessary calls to the 'updateUser' function if the 'name' is not a string.